mucommander-command/src/main/java/com/mucommander/command/CommandExtended.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of muCommander, http://www.mucommander.com
+ *
+ * muCommander is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * muCommander is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.mucommander.command;
+
+import javax.swing.Icon;
+
+/**
+ * Extended version of Command that also keeps optional icon.
+ */
+public class CommandExtended extends Command {
```

Comment:
Since base Command is more for those user/custom commands, I didn't want to. But if you think it is better to move it there, then I will move it.

Reply:
I think it would be better to add it there because let's say that we would do something similar to what this PR does for macOS also for Windows, we'll need to introduce another class that extends WindowsCmdCommand - it's ok to extend the use of the Command class in my perspective

Reply:
OK